detect_secrets/core/secrets_collection.py

```diff
@@ -119,6 +119,18 @@ def add_commit(self, commit: str) -> None:
             for secret in self.data[file]:
                 secret.commit = commit
 
+    def get_diff(self, newer: 'SecretsCollection') -> None:
```

Comment:
2 changes are needed:
1. type for `newer` variable should not be in quotes `'`
2. return type should not be None. since we are returning SecretsCollection. we should define the return type.

Reply:
That's how it's defined throughout the code, it actually returns me an error if I don't put it in quotes